core/templates/pages/learner-dashboard-page/progress-tab.component.html

```diff
@@ -1,9 +1,9 @@
 <div class="progress-tab">
-  <div class="skill-proficiency" [ngClass]="{'empty-section': emptySkillProficiency}" [ngStyle]="{'margin-top': windowIsNarrow? '35px': '64px'}" *ngIf="!windowIsNarrow || (windowIsNarrow && activeSubsection === LEARNER_DASHBOARD_SUBSECTION_I18N_IDS.SKILL_PROFICIENCY)">
+  <div tabindex="0" class="skill-proficiency" [ngClass]="{'empty-section': emptySkillProficiency}" [ngStyle]="{'margin-top': windowIsNarrow? '35px': '64px'}" *ngIf="!windowIsNarrow || (windowIsNarrow && activeSubsection === LEARNER_DASHBOARD_SUBSECTION_I18N_IDS.SKILL_PROFICIENCY)">
```

Comment:
```suggestion
  <div tabindex="0"
       class="skill-proficiency"
       [ngClass]="{'empty-section': emptySkillProficiency}"
       [ngStyle]="{'margin-top': windowIsNarrow? '35px': '64px'}"
       *ngIf="!windowIsNarrow || (windowIsNarrow && activeSubsection === LEARNER_DASHBOARD_SUBSECTION_I18N_IDS.SKILL_PROFICIENCY)">
```
this makes html files easily readible. Ditto elsewhere if the sentence is more than 80 chars

Reply:
Done